src/Sqs/Result/ReceiveMessageResult.php

```diff
@@ -76,7 +76,7 @@ protected function populateResult(ResponseInterface $response, HttpClientInterfa
 
                                     return $items;
                                 })($item->Value->BinaryListValue),
-                                'DataType' => ($v = $item->Value->DataType) ? (string) $v : null,
+                                'DataType' => (string) $item->Value->DataType,
```

Comment:
This belongs to `MessageAttributeValue`. It says that the DataType property is required. But it can be null. 

Will this work?

Reply:
No.. this will never be null

Reply:
Sorry, I was wrong. 

The `Result\MessageAttributeValue::$dataType` is not nullable. I was looking at the input.